src/test/End.t.sol

```diff
@@ -0,0 +1,834 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+pragma solidity ^0.8.13;
+
+import "dss-test/DSSTest.sol";
+import "ds-value/value.sol";
```

Comment:
Can we remove this dependency and just implement a simple contract here?
First to avoid the dependency itself, secondly because this one in particularly adds a warning when compiling.